src/components/date/date.pw.tsx

```diff
@@ -0,0 +1,1352 @@
+    });
+  });
+
+  [
+    ["chevron_right", "next"],
+    ["chevron_left", "previous"],
+  ].forEach(([arrow, month]) => {
+    test(`should use ${arrow} arrow in DayPicker to verify ${month} month is shown`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DateInputCustom />);
+
+      const input = getDataElementByValue(page, "input");
+      await input.fill(DATE_INPUT);
+      const inputParent = getDataElementByValue(page, "input").locator("..");
+      await inputParent.click();
+      const arrowElement = getDataElementByValue(page, arrow);
+      await arrowElement.click();
+      const pickerHeading = dayPickerHeading(page);
+      if (month === "next") {
+        await expect(pickerHeading).toHaveText(NEXT_MONTH);
+      } else if (month === "previous") {
+        await expect(pickerHeading).toHaveText(PREVIOUS_MONTH);
+      }
+    });
+  });
+
+  arrowKeys.forEach((key) => {
+    test(`should not change the displayed month when ${key} is pressed and next button is focused`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DateInputCustom />);
+
+      const inputParent = getDataElementByValue(page, "input").locator("..");
+      await inputParent.click();
+      const rightArrow = getDataElementByValue(page, "chevron_right").locator(
+        ".."
+      );
+      await rightArrow.focus();
+      await rightArrow.press(key);
+      const pickerHeading = dayPickerHeading(page);
+      await expect(pickerHeading).toHaveText("May 2022");
+    });
+  });
+
+  arrowKeys.forEach((key) => {
+    test(`should not change the displayed month when ${key} is pressed and previous button is focused`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DateInputCustom />);
+
+      const inputParent = getDataElementByValue(page, "input").locator("..");
+      await inputParent.click();
+      const rightArrow = getDataElementByValue(page, "chevron_left").locator(
+        ".."
+      );
+      await rightArrow.focus();
+      await rightArrow.press(key);
+      const pickerHeading = dayPickerHeading(page);
+      await expect(pickerHeading).toHaveText("May 2022");
+    });
+  });
+
+  test(`should allow a user to tab into the picker and through its controls`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DateInputCustom value="12/12/2022" />);
+
+    await page.focus("body");
+    await page.keyboard.press("Tab");
+    const input = getDataElementByValue(page, "input");
+    await expect(input).toBeFocused();
+    await page.keyboard.press("Tab");
+    const arrowLeft = getDataElementByValue(page, "chevron_left").locator("..");
+    await expect(arrowLeft).toBeFocused();
+    await page.keyboard.press("Tab");
+    const arrowRight = getDataElementByValue(page, "chevron_right").locator(
+      ".."
+    );
+    await expect(arrowRight).toBeFocused();
+    await page.keyboard.press("Tab");
+    const dayPicker = page.locator(".DayPicker-Day--selected");
```

Comment:
```suggestion
    const dayPicker = page.locator(`${DAY_PICKER_PREFIX}selected`);
```

**suggestion** (non-blocking): I'd missed this in the first review, but since you've bothered making a `DAY_PICKER_PREFIX` const we might as well use it wherever possible (you only seem to have used it in one place).

There are a few more places where this can be used too, just search for `.DayPicker-Day--` (I won't comment on each one individually!)